examples/image_streamer/golden_image.pp

```diff
@@ -57,7 +57,8 @@
   require => Image_streamer_golden_image['golden_image_4'],
   data    => {
     name                 => 'Golden_Image_2',
-    details_archive_path => 'log_archive.zip'  # can be either an absolute or relative path
+    details_archive_path => 'log_archive.zip',  # can be either an absolute or relative path
+    force                => true
```

Comment:
Should add a comment explaining what `force` does. Also should be included in resources.md